rs/nns/governance/canbench/canbench_results.yml

```diff
@@ -73,14 +73,14 @@ benches:
     total:
       calls: 1
       instructions: 66319497
-      heap_increase: 3
+      heap_increase: 5
       stable_memory_increase: 0
     scopes: {}
   list_neurons_stable:
     total:
       calls: 1
       instructions: 67632700
-      heap_increase: 4
```

Comment:
Is this increase expected?  I'm not sure I would have expected this to be affected (I just started looking at the PR, so maybe it will make sense soon)